lib/clickhousex/error.ex

```diff
@@ -36,16 +669,7 @@ defmodule Clickhousex.Error do
     end
   end
 
-  defp translate("57"), do: :table_already_exists
-  defp translate("60"), do: :base_table_or_view_not_found
-  defp translate("62"), do: :syntax_error
-  defp translate("81"), do: :database_does_not_exists
-  defp translate("82"), do: :database_already_exists
-  defp translate("28000"), do: :invalid_authorization
-  defp translate("08" <> _), do: :connection_exception
-  defp translate(code), do: code
-
-  defp get_constraint_violations(_reason) do
-    []
+  for {code, error} <- @clickhouse_codes do
+    defp translate(unquote(code)), do: unquote(String.to_atom(String.downcase(error)))
```

Comment:
Why do we need to know the translation? For fingerprinting, the code should be enough, and then there is the translation and full error message in `message`...

Reply:
You're right, I'll remove it then

Reply:
@ruslandoga turns out they're useful for tests

```
assert {:error, %{code: :table_already_exists}} ..
```

is better than

```
assert {:error, %{code: "67"}} ..
```

and certainly better than asserting random error text clickhouse returns. WYT?

Reply:
The old codes were wrong, so at least that's some improvement haha

Reply:
Yeah, asserts are definitely better :) We'd need to keep an eye on ClickHouse adding new errors then.